day2/day2.py

```diff
@@ -0,0 +1,65 @@
+from collections import Counter
+
+def count_letters(line: str) -> Counter:
+    return Counter(line)
+
+
+def does_repeat_x_times(line: str, x: int) -> bool:
+    letter_count = count_letters(line)
+    for letter in letter_count:
+        if letter_count[letter] == x:
+            return True
+    return False
+
+def checksum(data) -> int:
+    count_2 = 0
+    count_3 = 0
+
+    for line in data:
```

Comment:
Again, you can use map/filters -> http://book.pythontips.com/en/latest/map_filter.html map over each line with "count_letters" than do 2 filters -> one to select the ones with 2 and one to select 3, then do length(filtered(does_repeat_2) * length(filtered(does_repeat_3))